apiserver/facades/client/applicationoffers/state.go

```diff
@@ -39,52 +40,66 @@ func (pool statePoolShim) Get(modelUUID string) (Backend, func(), error) {
 	closer := func() {
 		releaser()
 	}
-	return &stateShim{st}, closer, nil
+	return &stateShim{st: st, Backend: commoncrossmodel.GetBackend(st)}, closer, nil
```

Comment:
It's needed because there's 2 lots of state interfaces (the common one and the one in this package) and they conflict. We have the same issues elsewhere too. It's messy.